mmv1/third_party/terraform/resources/resource_compute_security_policy.go.erb

```diff
@@ -271,6 +271,29 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 										Description: `Rate limit key name applicable only for the following key types: HTTP_HEADER -- Name of the HTTP header whose value is taken as the key value. HTTP_COOKIE -- Name of the HTTP cookie whose value is taken as the key value.`,
 									},
 
+									<% unless version == 'ga' -%>
+									"enforce_on_key_configs": {
+										Type:        schema.TypeList,
+										Description: `Enforce On Key Config of this security policy`,
```

Comment:
This field is not updatable now, so "ForceNew: true" can be added for this field.

I cut a ticket for the service team https://b.corp.google.com/issues/267376781

Reply:
Done :D Thank you so much for help us ❤️